pkg/kubelet/runonce.go

```diff
@@ -91,8 +91,12 @@ func (kl *Kubelet) runOnce(pods []*v1.Pod, retryDelay time.Duration) (results []
 		res := <-ch
 		results = append(results, res)
 		if res.Err != nil {
-			// TODO(proppy): report which containers failed the pod.
-			glog.Infof("failed to start pod %q: %v", format.Pod(res.Pod), res.Err)
+			failcontainername, err := kl.getFailedContainers(res.Pod)
+			if err != nil {
+				glog.Infof("failed to getFailContainers for pod %q: %v, error:%v", format.Pod(res.Pod), res.Err, err)
```

Comment:
Capitalize the log message: `Failed to getFailedContainers ...`

Reply:
Done